overlord/devicestate/handlers_install.go

```diff
@@ -92,7 +92,17 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 	}
 
 	// configure the run system
-	if err := sysconfigConfigureRunSystem(&sysconfig.Options{}); err != nil {
+	opts := &sysconfig.Options{}
+	cloudCfg := filepath.Join(dirs.RunMnt, "ubuntu-seed/cloud.cfg.d")
```

Comment:
looking at ubuntu-image we might need to support dirs that also go to /var and not only /etc. So now I don't we want this dir at top-level like that